packages/contracts/as/NFT/assembly/models.ts

```diff
@@ -58,15 +58,29 @@ export class Media {
     }
 }
 
+@nearBindgen
 export class NFTOnApprovedArgs {
     token_id: string
     owner_id: string
     approval_id: number
     msg: string
 }
 
+@nearBindgen
+export class StorageBalance {
+    total: string
+    available: string
+}
+
+@nearBindgen
+export class StorageBalanceBounds {
+    min: string
+    max: string
+}
+
 export const designs = new PersistentUnorderedMap<AccountId, Media>('md')
 export const owners = new PersistentSet<AccountId>('onrs')
+export const accounts = new PersistentUnorderedMap<AccountId, u128>('accs')
```

Comment:
I think this was in the `FT` contract to save the number of tokens owned by each user. Our contract is an `NFT` one 😄